widgets/blueprintCatalog/src/RepositoryCatalog.tsx

```diff
@@ -113,7 +114,7 @@ const RepositoryCatalog: FunctionComponent<RepositoryViewProps> = ({
             <div key={id}>
                 <StyledDataSegment
                     selected={isSelected}
-                    onClick={(event: Event) => {
+                    onClick={(event: React.MouseEvent) => {
```

Comment:
I think
```suggestion
                    onClick={event => {
```
should be enough as the type is already provided by `SegmentItemProps`.

You can remove `import React from 'react';` line then.

Reply:
Adjusted within cf09458bca99800fe4087edd646bb04e3aeb70bf 🎉